x-pack/plugins/search_indices/public/analytics/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export enum AnalyticsEvents {
```

Comment:
do we have any patterns we should follow here for naming events?

Reply:
no pattern established. One to think about but it looks fine as is.